projects/gloo/pkg/servers/iosnapshot/resources.go

```diff
@@ -24,8 +24,10 @@ var (
 		gloov1.ProxyGVK,
 	}
 
-	// PolicyGVKs is the set of GVKs that are used by a classic Gloo Gateway installation,
-	// when only the Edge Gateway APIs are being used
+	// PolicyGVKs is the set of GVKs that are used by a classic Gloo Gateway installation.
+	// This is the common set of GVKs that are available when only Edge Gateway APIs are being
```

Comment:
```suggestion
	// This is the common set of GVKs that are available when Edge Gateway APIs are being
```
i would remove the word `only`